src/jit/emitxarch.cpp

```diff
@@ -3670,7 +3785,9 @@ void emitter::emitIns_R_I(instruction ins, emitAttr attr, regNumber reg, ssize_t
             {
                 if (IsSSEOrAVXInstruction(ins))
                 {
-                    sz = 5;
+                    sz = emitInsSize(insCodeMI(ins));
```

Comment:
This should probably be a call to `emitInsSizeRR(id, insCodeMI(ins), val)`, but it would have required more refactorings to properly initialize `id` before these calls.